src/app/portlets/dot-templates/dot-template/dot-template.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { DotRouterService } from '@services/dot-router/dot-router.service';
+import { DotTemplatesService } from '@services/dot-templates/dot-templates.service';
+import { DotContainer } from '@shared/models/container/dot-container.model';
+import { Subject } from 'rxjs';
+import { take, takeUntil } from 'rxjs/operators';
+import * as _ from 'lodash';
+import { DotTemplate } from '@portlets/dot-edit-page/shared/models';
+
+@Component({
+    selector: 'dot-template-test',
+    templateUrl: './dot-template.component.html',
+    styleUrls: ['./dot-template.scss']
+})
+export class DotTemplateComponent implements OnInit, OnDestroy {
+    group: FormGroup;
+    editor: any;
+    isDisabled = true;
+    private destroy$: Subject<boolean> = new Subject<boolean>();
+    private originalValue: DotTemplate;
```

Comment:
This is not a `DotTemplate ` because is the value of the `group` and that only have `title, friendlyName and body` and DotTemplate have way more things.

Reply:
We'll leave it as is for now.